hs-bindgen/src-internal/HsBindgen/Config.hs

```diff
@@ -46,7 +48,8 @@ data Config_ path = Config {
   , programSlicing  :: ProgramSlicing
 
     -- * Backend
-  , haddockPathStyle :: PathStyle
+  , haddockPathStyle    :: PathStyle
+  , fileOverwritePolicy :: FileOverwritePolicy
```

Comment:
This should probably not be part of the common configuration, because it is unused by TH and Literate mode.